.gitlab/ci/.gitlab-ci.sdk-nightly.yml

```diff
@@ -21,6 +21,71 @@
   - demisto-sdk upload -i Packs/HelloWorld/Classifiers/classifier-mapper-incoming-HelloWorldTest.json --insecure
   - section_end "Upload Entities to Cortex XSOAR"
 
+.run_test_content: &run_test_content
+  - section_start "Run test-content" --collapsed
+  - |
+    if [[ -f "${ARTIFACTS_FOLDER_SERVER_TYPE}/conf.json" ]]; then	
+      cp "${ARTIFACTS_FOLDER_SERVER_TYPE}/conf.json" Tests/conf.json
+    fi
+  - |
+    ./Tests/scripts/run_tests.sh || EXIT_CODE=$?
+  - rm -f Tests/conf.json
+  - section_end "Run test-content"
+
+.run_end_to_end_tests: &run_end_to_end_tests
+  - section_start "End to End Tests"  --collapsed
+  - |
+    mkdir $ARTIFACTS_FOLDER/demisto-sdk
+    git clone -b ${SDK_REF} --single-branch --depth 1 https://github.com/demisto/demisto-sdk.git $ARTIFACTS_FOLDER/demisto-sdk
```

Comment:
and it's better to use:
`
https://github.com/<username>/<repository>/archive/<branch>.zip
`
and unzip it, as you don't need the heavy .git folder.

Reply:
I can change it if you think that it is worth the effort

Reply:
@MLainer1  can wait, if you can open a ticket on it